ADOPTERS.md

```diff
@@ -27,62 +27,63 @@ updating this list.
-| [UiPath](https://www.uipath.com/) | [@mjnovice](https://github.com/mjnovice) | Control plane for infrastructure management which powers [AutomationSuite](https://docs.uipath.com/automation-suite/automation-suite/2023.10/installation-guide-eks-aks/automation-suite-on-eksaks-overview) |
-| [SpareBank 1 Utvikling](https://sparebank1.dev/) | [@chlunde](https://github.com/chlunde) | Crossplane powers our Internal Developer Platform. It is utilized for day-to-day operations via GitOps and enabled us to execute a large-scale self-service migration of over a thousand production microservices, databases and caches from on-premises to EKS and managed AWS services. |
-| [Veset](https://veset.tv/) | [@pblgomez](https://github.com/pblgomez) | At Veset we are deploying all our backend resources in production environments to be managed by Crossplane. |
-| [Hyland Software](https://www.hyland.com/) | [@sethfduke](mailto:seth.duke@hyland.com) | Hyland is utilizing Crossplane to simplify the infrastructure provisioning process for internal development teams, providing a simple means of creating resources, while maintaining control over compliance, security, and best-practices of these resources through a suite of Compositions. |
+| Organization | Contact | Description of Use                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| ------------ | ------- |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
All the other lines of this file were updated - did this come from an automatic formatting tool you have installed in your environment maybe? wanting to make sure nothing else was edited about each of the lines showing up in the diff 🤔

Reply:
Ah yes, I'll clean that up & amend commit.

Reply:
Fixed, `0ded6a1` reflects the single line addition.